.github/workflows/qgsquick.yml

```diff
@@ -10,6 +10,10 @@ jobs:
       - name: download qgis
         run: |
           git clone https://github.com/qgis/QGIS.git --depth 1
+          cd QGIS
+          git fetch --depth 1 origin 1d17bf5bd35d7872f53c8e1c8b0a1e371616bf07
```

Comment:
can we have this `1d17bf5bd35d7872f53c8e1c8b0a1e371616bf07` as env variable on top of the file?